crv/crvQuality.cc

```diff
@@ -638,7 +638,7 @@ int Quality3D::computeJacDetNodes(apf::MeshEntity* e,
       interNodes[index] = apf::getDV(me,xi[index]);
       if(interNodes[index] < 1e-10){
         apf::destroyMeshElement(me);
-        return i+8;
+        return edge+8;
```

Comment:
Downstream code (eg https://github.com/SCOREC/core/blob/master/crv/crvShape.cc#L55) relies on this numbering/marking scheme.